src/main/java/io/codelex/flightplanner/FlightService.java

```diff
@@ -0,0 +1,91 @@
+package io.codelex.flightplanner;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Service
+public class FlightService {
+    private long id = 0;
+    private final FlightRepository flightRepository;
+
+    public FlightService(FlightRepository flightRepository) {
+        this.flightRepository = flightRepository;
+    }
+
+    public void clearFlights() {
+        flightRepository.clearFlights();
+    }
+
+    public PageResult searchFlightByFromToAndDepartureDate(SearchFlightsRequest flight) {
+        return flightRepository.searchFlightByFromToAndDepartureDate(flight);
+    }
+
+    public List<Airport> searchAirport(String search) {
```

Comment:
Šis. tagad ir OK, bet lai vēlāk atvieglotu Tev dzīvi - pārnes šīs metodes funkcionalitāti uz repository, jo šo darbību varēs izdarīt datu bāze. (Līdzīgu komentāru pievienošu citām metodēm, kurām saturs būtu jāpārnes uz repository)